credentials.js

```diff
@@ -0,0 +1,5 @@
+const key = {
+  NEWS_API_KEY: "9325639d44f94e168a54898f6243b3d3",
+  NEWS_API_KEY_2: "7cf0171f9a004bd386992a39ec8f8fd6"
+};
+export default key;
```

Comment:
I don't think its safe to leave this Api keys out here, you should leave them empty, add this file into the `.gitignore` and just change it locally